adapters/adapter.py

```diff
@@ -485,16 +485,9 @@ def uploadOSMService(self,request):
             LOG.debug(token)
             #file_to_upload = content['service']
             file_to_upload = request
-            file_composed = "@" + file_to_upload
```

Comment:
@bobeal Are you sure this is not used?

Reply:
yes, when uploading the NSD, only the `file_to_upload` variable is used.